projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -243,7 +256,7 @@ export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
     return beginning + ' - ' + ending;
   }
 
-  setDisplayData(): any[] {
+  getDisplayData(): any[] {
```

Comment:
I just updated this name because I thought it was a bit more semantically appropriate since this returns a value rather than setting a component property or something, and this matches the pattern of the methods below.